linux/guest_customization/check_gosc_support_status.yml

```diff
@@ -107,6 +120,7 @@
               with_items: "{{ gosc_matrix_os_versions }}"
 
```

Comment:
Miracle Linux 8.x VERSION_ID in /etc/os-release is not precise, it only has major version. I suggest you update `linux/utils/get_linux_system_info.yml` to get the Miracle LInux 8.x's minor version from /etc/os-release and update values of `guest_os_ansible_distribution_ver` and `guest_os_ansible_distribution_minor_ver`

Reply:
Pass to test the code change:

ok: [localhost] => {
    "msg": [
        "Guest OS: MIRACLE",
        "Guest OS type: Linux",
        "Guest OS arch: x86_64",
        "Guest OS bit: 64-bit",
        "Guest OS major version: 8",
        "Guest OS minor version: 8",
        "Guest OS version: 8.8",
        "Guest OS kernel: 4.18.0-477.10.1.el8_8.x86_64",
        "Guest OS release: Peony",
        "Guest OS edition: ",
        "Guest OS family: RedHat",
        "Guest OS with desktop environment: True",
        "Guest OS display manager: gdm"
    ]
}